spec/lib/tasks/populate_postcodes_table_spec.rb

```diff
@@ -1,12 +1,11 @@
 require "rake"
 require "spec_helper"
 
-Rails.application.load_tasks
-
-RSpec.describe "Tasks" do
+RSpec.describe "populate_postcodes_table task" do
   describe "when running the task" do
     before do
       Postcode.destroy_all
+      allow_any_instance_of(IO).to receive(:puts) # stop command line messages appearing in test output
```

Comment:
Minor: instead of stubbing out IO, we could wrap the rake task using: `expect { <call> }.to output.to_stdout` if we expect output which is a bit nicer

Reply:
Done!